.github/workflows/lint.yaml

```diff
@@ -0,0 +1,19 @@
+name: Lint
```

Comment:
We could also add Build CI workflow plus Tests once unit tests are added

Reply:
done